.storybook/test-runner.ts

```diff
@@ -1,13 +1,23 @@
 import type { TestRunnerConfig } from '@storybook/test-runner';
 
+const STORYBOOK_ELEMENT_ID = '[data-test-id="story-root"]';
+
 const config: TestRunnerConfig = {
-  postRender: async (page, context) => {
-    // Gather the page HTML inner content for a DOM HTML Snapshot
-    const rootElementId = '[data-test-id="story-root"]';
-    const rootElement = await page.locator(rootElementId);
+  postRender: async page => {
+    // We wait the page for loading for at least one second
```

Comment:
```suggestion
    // We wait for the page to load for at least one second
```

Reply:
I can still apply this suggestion if you’d like 👀